Assets/BossRoom/Scripts/Client/Game/Character/ClientCharacterVisualization.cs

```diff
@@ -242,8 +242,11 @@ void Update()
             }
 
             // NetworkTransform is interpolated - we can just apply it's position value to our visual object
-            transform.position = m_PhysicsWrapper.Transform.position;
-            transform.rotation = m_PhysicsWrapper.Transform.rotation;
+            VisualUtils.SmoothMove(transform,
```

Comment:
What does smoothmove provide that Vector3.lerp doesn't?

Reply:
I tried to locally reproduce the same effect with Vector3.Lerp and it's definitely not as snappy as SmoothMove(). The slow ins and outs are really pronounced with Vector3.Lerp. I can create something similar, however, with Vector3.SmoothDamp, but I'm just cautious of introducing this since the SmoothMove was finely tuned for the actions.

Reply:
as discussed, let's try to use lerp with attribute